collect_stata/read_stata.py

```diff
@@ -34,17 +32,16 @@ def cat_values(varscale, data):
     return cat_list
 
 
-def scale_var(var, varscale, datatable):
-    """
-    Select vartype
+def scale_var(var: str, varscale: dict, datatable) -> str:
+    """Select vartype
 
-    Input:
-    var: string
-    varscale: dict
-    datatable: pandas DataFrame
+    Args:
+        var (string): Name of the variable.
```

Comment:
You already used the type hint above, so you could omit it here.
If you want to repeat the type here, please use "str".